content/admin.py

```diff
@@ -62,6 +62,16 @@ class DonateSectionAdmin(admin.ModelAdmin):
     fields = ['category','order', 'color', 'title', 'img', 'text']
     list_display = ('order', 'category', 'title')
 
+class SubMenuInline(admin.TabularInline):
+    model = SubMenu
+    extra = 3
+
+
+class MenuAdmin(admin.ModelAdmin):
+    list_display = ('title', 'order')
+    fieldsets = [(None,{'fields': ['title']}),]
```

Comment:
when you leave comments can you please be more specific. There is a space and there shouldn't be, or there should be and there isn't?